examples/backbone_marionette/css/app.css

```diff
@@ -7,3 +7,7 @@
 #footer {
 	display: none;
 }
+
+#clear-completed::after {
+	top: 0;
+}
```

Comment:
It happens because the `Clear completed` text is outputted with a lot of whitespace around it. That should be fixed rather than hacking around it with CSS.